create_fail_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
`bash` is not installed on all systems. Since this is such a small script, I recommend removing the `local` in the declaration for the function variables and changing the shebang to the following for maximum portability:
```suggestion
#!/bin/sh
```